roles/openshift_label_nodes/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Get Nodes Config
+  shell: osc get nodes -o json | sed -e '/"resourceVersion"/d'
+  register: output
+
+- name: Set node regions
+  set_fact:
+    node_facts: "{{ output.stdout | from_json
+                  | oo_set_node_label('region', {{oln_node_region}}) }}"
+
+- name: Set infra node labels
+  set_fact:
+    node_final_facts: "{{ node_facts
+                  | oo_set_node_label('region', {{ocom_infra_node_label}}, {{oln_infra_node_filter_key}}, {{oln_infra_node_filter_value}}) }}"
+
+- name: Write node config to temp file
+  copy:
+    content: "{{ node_final_facts }}"
+    dest: /tmp/nodes.json
```

Comment:
Also, initial node labeling can be done by setting the openshift_node_labels var, I'm not sure if it is working in the current master branch, but it's working in my configTemplates branch that is going in for beta4 (it also includes a default master scheduler.json, though it needs to be templated further for GA)